conjure-java-core/build.gradle

```diff
@@ -43,7 +43,7 @@ dependencies {
     compile 'com.fasterxml.jackson.core:jackson-databind'
     compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
     compile 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8'
-    compile 'com.google.googlejavaformat:google-java-format'
+    compile 'com.palantir.javaformat:palantir-java-format'
```

Comment:
code change

Reply:
I didn't realize that we service loaded the formatter. That's cool!